helm/kanister-operator/templates/rbac.yaml

```diff
@@ -16,8 +16,10 @@ rules:
   - ""
   resources:
```

Comment:
@PrasadG193 Should we use * here? Since we claim create and delete of **Kubernetes resources** on docs.

Reply:
@shlokchaudhari9 in my opinion, we should not give access by default. If someone wants to do that, they can update the clusterroles accordingly. We can add a note in docs for that